### DISCUSSION TITLE
Config Page Blank?

### DISCUSSION THREAD

**Discussion by CamdenJaxx** [JulianPrieber/littlelink-custom#427] [Issues]
Hey there. Fresh install, everything looks great, but the config page simply loads a blank section for the main content, nothing else. 

Screenshot: https://i.gyazo.com/0dd62a02f699023f7ab159d81bb8a53e.jpg

**Comment by JulianPrieber** [comment]
Any errors in the config?

**Reply by CamdenJaxx** [reply]
I've checked and don't see any error logs being created anywhere. Nothing is being output to indicate the error that I can see. I double checked and debug is enabled, so I would imagine it should present itself somewhere along the lines.

**Reply by JulianPrieber** [reply]
In your browser, in the developer console

**Reply by CamdenJaxx** [reply]
No, no issues anywhere in the console either.

**Reply by JulianPrieber** [reply]
Ah, okay.
Is this the docker version or the web version?

**Reply by CamdenJaxx** [reply]
This is for the web version.

**Reply by JulianPrieber** [reply]
Could you give me some details about your setup?

**Reply by CamdenJaxx** [reply]
Working with a E3-1230 v2, 16GB Ram, running CentOS7 and PHP version 8.0.28. If you want specific information, please let me know as I'm not aware of what's best to include to best answer your query.

**Reply by JulianPrieber** [reply]
What about the web server type? NGINX, Apache...

**Reply by CamdenJaxx** [reply]
Apologies for the delayed response. Great news - I did a few system updates and everything works just fine now. For the record, I am using Litespeed 1.7.16 through CyberPanel. After the update to the system and to CyberPanel (bringing both current), all is working just fine. The config page loads without any issue. Many thanks for your time, patience, and assistance!

**Reply by JulianPrieber** [reply]
👍
